spring-cloud-stream-app-documentation-maven-plugin/src/main/java/org/springframework/cloud/stream/app/documentation/plugin/ConfigurationMetadataDocumentationMojo.java

```diff
@@ -89,7 +90,8 @@ public void execute() throws MojoExecutionException {
 			}
 
 			ScatteredArchive archive = new ScatteredArchive(mavenProject);
-			ClassLoader classLoader = metadataResolver.createClassLoader(archive);
+			BootClassLoaderFactory bootClassLoaderFactory = new BootClassLoaderFactory(archive, null);
```

Comment:
The `BootApplicationConfigurationMetadataResolver` also creates a `classloader` when it lists properties again. Can we extract the classloader from the `BootApplicationConfigurationMetadataResolver` object so that we can use here as well?